src/components/Widgets/useWidgetTheme.ts

```diff
@@ -42,7 +42,8 @@ export const useWidgetTheme = (): PartnerTheme => {
           },
           secondary: {
             // FIXME: we need to find out how to use the correct color from the main theme config
-            main: darkTheme.palette.accent2.main,
+            // main: darkTheme.palette.accent2.main,
+            main: '#ff0420',
```

Comment:
You were right about those colors which are used from the Theme passed into ThemeProvider, let´s keep theme.palette.secondary.main here?